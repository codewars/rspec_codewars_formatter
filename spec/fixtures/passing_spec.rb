RSpec.describe "Passing" do
  it "passes" do
    expect(true).to be_truthy
  end
end
