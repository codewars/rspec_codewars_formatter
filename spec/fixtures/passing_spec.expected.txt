
<DESCRIBE::>Passing

<IT::>passes

<PASSED::>Test Passed

<COMPLETEDIN::>0.9258

<COMPLETEDIN::>1.1349
